DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -110,7 +110,11 @@ h1#page-heading {
 }
 
 .responsive-iframe-wrapper-padding {
-  padding-bottom: nhsuk-spacing(7);
+  padding-bottom: nhsuk-spacing(8);
+}
+
+.responsive-iframe-wrapper-extra-padding {
+  padding-bottom: nhsuk-spacing(8) * 2;

```

Comment:
I'd recommend pulling nhsuk-spacing(8) out into a variable at the top of this file, so that people don't miss one of them when changing the value.
Something like `$responsive-iframe-padding: nhsuk-spacing(8)`

Reply:
Good idea, I've pulled it out into a variable called $iframe-padding-bottom